tests/framework/microvm.py

```diff
@@ -144,18 +144,31 @@ def __init__(
         # External clone/exec tool, because Python can't into clone
         self.bin_cloner_path = bin_cloner_path
 
+        # Flag checked in destructor to see abnormal signal-induced crashes.
+        self.expect_kill_by_signal = False
```

Comment:
Wouldn't it make sense to have this always on?

Reply:
No.. default in tests and here is clean exit and not being killed by a signal.

Some tests validate/expect the microvm to be killed by a signal, this flag is set to `True` by those tests so microvm destructor knows to not consider it an error.